docs/source/intro.rst

```diff
@@ -0,0 +1,44 @@
+===========
+ome-zarr-py
+===========
+
+Experimental support for multi-resolution images stored in Zarr filesets, according to the `OME zarr spec`_.
+
+
+Features
+--------
+
+- Use as a image reader plugin for `napari`_. See `napari-ome-zarr`_.
```

Comment:
When I went to napari-ome-zarr as instructed, there is no mention of ome-zarr-py there. Confusing ?

Reply:
```suggestion
- Use as an image reader plugin for `napari`_. See `napari-ome-zarr`_.
```

fix typo